### PR TITLE
chore(refactor): Improving the code quality

### PR DESCRIPTION
**Pull request by kushthedude**
Refers #3735

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110)

**Comment by iamareebjamal** [comment]
12 issues are fixed in codacy and 144 files changed. Some changes are unneeded. Like the ones adding newlines between component properties. Try to keep PR as short as the issues it is fixing

https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110

**Comment by kushthedude** [comment]
> 12 issues are fixed in codacy and 144 files changed. Some changes are unneeded. Like the ones adding newlines between component properties. Try to keep PR as short as the issues it is fixing
> 
> https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110

Possible fixed issues 539

**Comment by kushthedude** [comment]
It has fixed the issues of spaces and indents which are required b/w class members. I have also upgraded the Ember-Suave Version and also have turned off some rules like `no-await-in-loop`. There are currently 3700+ issues in Codacy and this PR fixes around 1600 of them, Others are no-get properties which will be coming in next PR. Hoping the rating will change after this

**Comment by iamareebjamal** [comment]
spaces between component methods make sense. Between component properties, not so much. Also, this adds line break after every if statement as well which is really unnecessary

**Comment by kushthedude** [comment]
> spaces between component methods make sense. Between component properties, not so much. Also, this adds line break after every if statement as well which is really unnecessary

It was generated as an automated fix. I will check if reverting those spaces doesn't generate any warning, I will do.

**Comment by kushthedude** [comment]
> if
```js
29:3  error  Expected blank line between class members  lines-between-class-members
```

When I remove the spaces

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110)

File: app/controllers/events/view/edit/sessions-speakers.js [hunk 1]
```diff
@@ -28,4 +28,4 @@ export default Controller.extend(EventWizardMixin, {
       );
     }
   }
-});
\ No newline at end of file
```
Comment (niranjan94) [inline app/controllers/events/view/edit/sessions-speakers.js]: Codacy Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110)

File: app/controllers/events/view/edit/attendee.js [hunk 2]
```diff
@@ -38,4 +39,4 @@ export default Controller.extend(EventWizardMixin, {
       }
     }
   }
-});
\ No newline at end of file
```
Comment (niranjan94) [inline app/controllers/events/view/edit/attendee.js]: Codacy Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204&bid=15826110)

**Comment by iamareebjamal** [comment]
Disable `padding-line-between-statements`

**Comment by iamareebjamal** [comment]
Or configure it properly https://eslint.org/docs/rules/padding-line-between-statements

File: .eslintrc.js [hunk 3]
```diff
@@ -19,6 +19,8 @@ module.exports = {
     'arrow-spacing': 'error',
     'no-var': 'error',
     'no-useless-escape': 'off',
+    'no-await-in-loop': 'off',
+    'import/no-relative-parent-imports': 'off',
```
Comment (CosmicCoder96) [inline .eslintrc.js]: Why are you turning validations off to improve code quality?

**Comment by kushthedude** [comment]
No relative parent import is failing for our config files. Also there are
places where we need await in loop.

**Comment by kushthedude** [comment]
Most of is solved and remaining will get solve when we merge the recent dependency PRs

**Pull request closed without merging**